src/components/RecipeForm/RecipeForm.js

```diff
@@ -0,0 +1,60 @@
+import './RecipeForm.css'
+import {Link} from 'react-router-dom'
+import { Component } from 'react';
+
+class RecipeForm extends Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      email: '',
+      password: '',
+    }
+  }
+  updateInput = (e) => {
+    let type = e.target.type
+    let value = e.target.value
+    this.setState({[type]: value})
+  }
+
+  submitForm = () => {
+    console.log('Submitted!')
+  }
+
+  render() {
+    return (
+      <div className="RecipeForm">
+        <h1>Let's contribute!</h1>
+        <p>We need a little information from you below to make a recipe and
+        connect it to a non profit organization you want to support!</p>
+        <form>
+          <label>
+            Recipe Name:
+            <input type='text' onChange={this.updateInput}/>
+          </label>
+          <label>
+            Recipe Description:
+            <input type='text' onChange={this.updateInput}/>
+          </label>
+          <label>
+            Ingredients
+            <input type='text' onChange={this.updateInput}/>
+          </label>
+          <label>
+            Steps
+            <input type='email' onChange={this.updateInput}/>
+          </label>
+          <label>
+            None Profit Organization
```

Comment:
I think I needed to see this, this was the format I had for the Full Recipe page and I was just guessing as to what information would be needed from the mockData that we have available.

Reply:
The mock data should be available already, but we also have some end points I am looking at connecting this weekend :)